.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "printWidth": 120,
+  "bracketSpacing": true,
+  "explicitTypes": "preserve",
+  "tabWidth": 2,
```

Comment:
this matches the prettier file in protocol

Reply:
I was just following Chris suggestion [from here](https://github.com/across-protocol/sdk-v2/pull/1#discussion_r821936666)